### PR TITLE
Use rightmost part for python callback naming

### PR DESCRIPTION
**Pull request by shakao**
We were previously concatenating the entire qname which resulted in controller.combos.attachCombo -> combos_controller.combos.attach_combo during the recursion.

Tested: Warehouese, Jumping Mateo, HackMan, Raptor Run

Fixes Warehouse and Jumping Mateo from https://github.com/microsoft/pxt-arcade/issues/2083

### CODE REVIEW

File: pxtpy/pydecompiler.ts [hunk 1]
```diff
@@ -934,11 +934,11 @@ namespace pxt.py {
             // TODO(dz): Impl skip namespaces properly. Right now we just skip the left-most part of a property access
```
Comment (darzu) [inline pxtpy/pydecompiler.ts:934]: I think this addresses this comment now.

**Comment by darzu** [comment]
If it's easy, want to add a test case?

**Comment by shakao** [comment]
i couldn't quite figure out how to get a test going since we only hit this case when we have the qname from the typechecker, otherwise we output generic my_function, my_function2 names; including the namespaces as part of the test file didn't work, let me know if i'm missing something obvious!

**Comment by abchatra** [comment]
Merging now for tomorrow testing. Feel free to add test case later.

**Pull request merged**